src/git/install.sh

```diff
@@ -235,7 +273,28 @@ echo "Downloading source for ${GIT_VERSION}..."
 curl -sL https://github.com/git/git/archive/v${GIT_VERSION}.tar.gz | tar -xzC /tmp 2>&1
 echo "Building..."
 cd /tmp/git-${GIT_VERSION}
-make -s USE_LIBPCRE=YesPlease prefix=/usr/local sysconfdir=/etc all && make -s USE_LIBPCRE=YesPlease prefix=/usr/local sysconfdir=/etc install 2>&1
+if [ "${ADJUSTED_ID}" = "alpine" ]; then
+    # ref. <https://github.com/alpinelinux/aports/blob/32ac93ffb642031b88ba8639fbb3abb324169dea/main/git/APKBUILD#L126>
+    git_options=("prefix=/usr")
+    git_options+=("NO_REGEX=YesPlease")
+    git_options+=("NO_GETTEXT=YesPlease")
+    git_options+=("NO_SVN_TESTS=YesPlease")
+    git_options+=("NO_SYS_POLL_H=1")
+    git_options+=("ICONV_OMITS_BOM=Yes")
+    git_options+=("INSTALL_SYMLINKS=1")
+    git_options+=("CC=${CC:-cc}")
+    git_options+=("CXX=${CC:-c++}")
+    # git_options+=("CFLAGS=$CFLAGS -flto=auto")
+    # git_options+=("LDFLAGS=$LDFLAGS -flto=auto")
+    git_options+=("USE_LIBPCRE2=YesPlease")
+    git_options+=("PYTHON_PATH=/usr/bin/python3")
+    git_options+=("perllibdir=$(_perl_config vendorlib)")
```

Comment:
Why do we need this? Le't remove this as well.

Let's set the options as close as possible to other distros for avoiding any confusion.